administrator/language/de-DE/com_guidedtours.sys.ini

```
@@ -0,0 +1,376 @@
+
+COM_GUIDEDTOURS_TOUR_TAGS_STEP_VERSIONNOTE_TITLE="Einen Versionshinweis hinzufügen"
+COM_GUIDEDTOURS_TOUR_TAGS_STEP_VERSIONNOTE_DESCRIPTION="Dies ist ein optionales Feld, um die Version dieses Schlagwortes zu identifizieren."
+
+COM_GUIDEDTOURS_TOUR_TAGS_STEP_SAVECLOSE_TITLE="'Speichern &amp; Schließen' wählen"
+COM_GUIDEDTOURS_TOUR_TAGS_STEP_SAVECLOSE_DESCRIPTION="Schlagwort speichern und schließen."
+
+COM_GUIDEDTOURS_TOUR_TAGS_STEP_CONGRATULATIONS_TITLE="Herzlichen Glückwunsch!"
+COM_GUIDEDTOURS_TOUR_TAGS_STEP_CONGRATULATIONS_DESCRIPTION="Ein neues Schlagwort wurde erstellt."
+
+COM_GUIDEDTOURS_TOUR_BANNERS_TITLE="Wie erstellt man Banner?"
+COM_GUIDEDTOURS_TOUR_BANNERS_DESCRIPTION="Diese Tour zeigt, wie man einen Banner erstellt."
+
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_NEW_TITLE="'Neu' wählen"
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_NEW_DESCRIPTION="Schaltfläche 'Neu' auswählen, um ein neues Banner zu erstellen."
+
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_TITLE_TITLE="Einen Titel für das Banner hinzufügen"
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_TITLE_DESCRIPTION="Den gewünschten Titel für das Banner eingeben."
+
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_ALIAS_TITLE="Einen Alias eingeben"
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_ALIAS_DESCRIPTION="Den internen Namen für diesen Banner eingeben. Wird dieses Feld leer gelassen, dann wird ein Standardwert in Kleinbuchstaben mit Bindestrichen anstelle von Leerzeichen als Teil der URL verwendet."
+
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DESCRIPTION_TITLE="Eine Beschreibung hinzufügen"
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_DESCRIPTION_DESCRIPTION="Eine Beschreibung für das Banner im Editor hinzufügen."
+
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_STATUS_TITLE="Status wählen"
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_STATUS_DESCRIPTION="Den Status des Banners auswählen."
+
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_CATEGORY_TITLE="Kategorie wählen"
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_CATEGORY_DESCRIPTION="Die Kategorie für dieses Banner auswählen. Es kann auch direkt eine neue Kategorie angelegt werden, indem man den Namen in das Feld eingibt und mit der Eingabetaste bestätigt."
+
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_PINNED_TITLE="Wichtig"
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_PINNED_DESCRIPTION="'Wichtig' auswählen, um dem Banner Vorrang vor nicht wichtigen Bannern zu geben."
+
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_VERSIONNOTE_TITLE="Einen Versionshinweis hinzufügen"
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_VERSIONNOTE_DESCRIPTION="Dies ist ein optionales Feld, um die Version dieses Banners zu identifizieren."
+
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_SAVECLOSE_TITLE="'Speichern &amp; Schließen' wählen"
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_SAVECLOSE_DESCRIPTION="Banner speichern und schließen."
+
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_CONGRATULATIONS_TITLE="Herzlichen Glückwunsch!"
+COM_GUIDEDTOURS_TOUR_BANNERS_STEP_CONGRATULATIONS_DESCRIPTION="Ein neues Banner wurde erstellt"
```

```suggestion
COM_GUIDEDTOURS_TOUR_BANNERS_STEP_CONGRATULATIONS_DESCRIPTION="Ein neues Banner wurde erstellt."
```
Fehlt im Core. Sollen wir den ergänzen und dann noch einen PR für den Core erstellen?

---

Ja

---

https://github.com/joomla/joomla-cms/pull/40401